go.mod

```diff
@@ -2,20 +2,24 @@ module github.com/NearNodeFlash/nnf-sos
 
 go 1.16
 
+replace github.com/NearNodeFlash/nnf-dm => ../nnf-dm
```

Comment:
Are you doing a circular dependency with nnf-dm and nnf-sos vendoring each other?
Let's not do that.

Reply:
Just the API